opendatalakehouse/README.md

```diff
@@ -11,8 +11,8 @@ For our tutorial, we will use a raw airlines dataset to -
 0. [Pre-reqs](00_prereqs.md) - Set up CDP user workload password and deploy the Applied Machine Learning Prototype (AMP) for `Canceled Flight Prediction`
 1. [Ingest](01_ingest.md) - Build an ingestion data pipeline to enable advanced analytics and Machine Learning (ML) use cases
 2. [Analyze](02_analyze.md) - Explore the ingested data and conduct an interactive analysis
-3. [Predict](03_predict.md) - Predict the likelihood of a flight being canceled based on historical records
-4. [Do More with Iceberg](04_iceberg.md) - Test Iceberg features such as Time Travel and Partition Evolution, and change the ML Project to train the `Canceled Flight Prediction` model using the Data Lakehouse (Iceberg) data
+3. [Predict](04_predict) - Predict the likelihood of a flight being canceled based on historical records
```

Comment:
Let us add one more line here for visualize here and add it to the list may be like 
`3. [Visualize](/03_visualize.md) - Create a visualization dashboard using cloudera Datawarehouse`

Reply:
lets rename headings in the first line of all readme like 03_predict -> 04_predict , 04_iceberg ->05_iceberg

https://github.com/cloudera/cdp-trial-patterns/pull/11/files#diff-ef7c5fa6647545ea0b54c52d1809bbc04bf41e7bf7b593bd8304225ef19f67eeR1